azure-sql/database/authentication-aad-overview.md

```diff
@@ -118,6 +118,10 @@ To create a contained database user in Azure SQL Database, Azure SQL Managed Ins
   - `SUSER_ID(<name>)`
   - `SUSER_SID(<name>)`
 
+- Azure SQL Database doesn't create implicit users for users logged in as part of an Azure Active Directory (Azure AD) group membership. Because of this, various operations that require assigning ownership will fail, even if the Azure AD group is added as a member to a role with those permissions.
```

Comment:
> Azure Active Directory (Azure AD)

@rwestMSFT Can we simplify this to just Azure AD?

Reply:
@nofield I didn't check the rest of the article to see if we'd referenced it as such, but sure. It looks like we do that earlier on.